quickstart-templates/workload-identity-creates-workload-identities/main.bicep

```diff
@@ -0,0 +1,54 @@
+targetScope = 'tenant'
+extension microsoftGraph
+
+resource azureRootManagementGroup 'Microsoft.Management/managementGroups@2023-04-01' existing = {
+  scope: tenant()
+  name: '<guid>'
+}
+
+@description('The owner of the organization that it assigned to a workload identity that is used by GitHub Actions to deploy further resources.')
+var gitHubOwner = '<MyOrganization>'
+
+@description('The GitHub repository that is assigned to a workload identity that is used by GitHub Actions to deploy further resources.')
+var gitHubRepo = '<MyOrganizationRootProject>'
```

Comment:
Should this be a `param`, to allow this to be set at the az cli/ps command line?